### ISSUE TITLE
update Planet Preparation Videos

### ISSUE THREAD

**Issue by sahilvunnam** [open-learning-exchange/open-learning-exchange.github.io#3416]
### Please verify the following:

- [X] This issue is related to the [Virtual Interns First Steps](https://open-learning-exchange.github.io/#!pages/vi/vi-first-steps.md) document or [Planet User Manual](https://open-learning-exchange.github.io/#!pages/manual/planet/overview.md).
- [X] You have searched for relevant keywords in [recent open and closed issues](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues?q=is%3Aissue+label%3Avi) to avoid duplicates.

### URL to the MDwiki page containing the problem

https://open-learning-exchange.github.io/#!./pages/vi/vi-planetapps.md

### Problem

I think updating the Planet Preparation Videos, that takes you through the function of your Planet, need to be updated. For one since the videos are outdated, most of what I learned about the website's features and it's UI was done solely through me going through. Obviously it is good that I'm forced to look through the website myself, but following an updated video is easier.

### Proposed solution(s)

Create new videos of a person going through each of the websites main pages such as library, courses, teams, meetups, etc. Also I think the recording should also have the person's face in it, to make it seem more personable.

**Comment by G201021** [comment]
It could also be helpful if they provide a video showcasing their myPlanet app on android. It is not in its finished state yet and there is a lot of room of improvement, but for the virtual interns and beta testers, to get around it, we need some solid explanation on what they wanna see in the final version of their app and future updates.

**Comment by xyb994** [comment]
Thank you for the suggestion. This is a considerable amount of work. If you’d like to proceed with it, I suggest selecting one of the videos to update.

**Comment by ssamikshaa** [comment]
I agree that videos showing the updated UI would help considerably, especially since much of the pages and navigation has changed. It would be nice to have videos covering the new community and enterprises pages as well as situations in which they are used.

**Comment by tuyisabeangelique** [comment]
Yes, updated Planet Preparation Videos would be very helpful for new users.

**Comment by munishpatel** [comment]
The new videos can be made eye catching, so the new users get attracted to the community!

**Comment by 3011089503** [comment]
The new video is very useful for new users!

**Comment by Sakshamnayyar** [comment]
Updated videos would help a lot. Along with that, maybe we could also add a feature like in-app tips or guided walkthroughs that highlight key features as users explore the platform. It could make onboarding even smoother.
